pkg/exposer/csi_snapshot.go

```diff
@@ -462,10 +464,14 @@ func (e *csiSnapshotExposer) createBackupPod(
 		VolumeSource: corev1.VolumeSource{
 			PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
 				ClaimName: backupPVC.Name,
-				ReadOnly:  true,
 			},
 		},
 	}}
+
```

Comment:
Line 459: `volumeMounts, volumeDevices, volumePath := kube.MakePodPVCAttachment(volumeName, backupPVC.Spec.VolumeMode, true)`

The last parameter is a bool for setting readOnly to `volumeMount`, since we set it in `volumeSource` conditionally, let's also set that one conditionally. Or change that line like:
volumeMounts, volumeDevices, volumePath := kube.MakePodPVCAttachment(volumeName, backupPVC.Spec.VolumeMode, backupPVCReadOnly)

Reply:
@Lyndon-Li yes, that's what @shubham-pampattiwar was suggesting too. I had initially not changed that since it doesn't seem to be required for SELinux or OpenShift. Without doing this, relabeling still works but the mounted volume is read-only. I can change this to match `backupPVCReadOnly`, although the effect is that Velero will be able to modify the filesystem there. If everyone is OK with that, then I'll make the change.

Reply:
@Lyndon-Li I made that change. It doesn't seem to break anything.

Reply:
I think it is OK, as this is the previous behavior without backupPVC readOnly feature.
Conclusively, without backupPVCReadOnly set, everything is the same with 1.14; with backupPVCReadOnly set to true, all places are set to readOnly.